src/pages/groupComparison/GroupComparisonPage.tsx

```diff
@@ -190,7 +207,7 @@ export default class GroupComparisonPage extends React.Component<IGroupCompariso
             }
             let ret;
             if (this.store.sessionClinicalAttributeName) {
-                ret = <span>{studyHeader}Groups from <span style={{color:"#3487c7"}}>{this.store.sessionClinicalAttributeName}</span></span>
+                ret = <span>{studyHeader}Groups from <span style={{fontWeight:"bold", fontStyle:"italic"}}>{this.store.sessionClinicalAttributeName}</span></span>
```

Comment:
i like <strong><i> but i'm not quite sure why.   i guess from the old days of "semantic HTML."   no change necessary